### ISSUE TITLE
Update package.json script paths?

### ISSUE THREAD

**Issue by xorander00** [hackmdio/codimd#491]
I noticed that your package.json files use the hard-code ./node_modules/.bin/<js-executable> path to invoke CLI scripts.

While you can do that, it's probably a lot better to just leave off the explicit path to the .bin/ directory.  npm automatically includes that directory when invoking commands as part of a script target.

Some commands, such as webpack or rollup, might be installed globally for platform-specific reasons.  If that's the case, letting npm use the global version if it doesn't find the local version would be a better option.

**Comment by SISheogorath** [comment]
PRs welcome :)

**Comment by ccoenen** [comment]
I tend to disagree, package.json defines exactly what's needed for _this_ project. This is what's installed in `node_modules`. If Webpack is needed, it should be in the package.json (and therefore in the `node_modules`).

Is there any authorative source that lays out a best practice?

**Comment by xorander00** [comment]
Ideally I would agree about specifying exact deps in package.json.  However, forcing the path takes away admin flexibility.  It's why NPM automatically checks local deps and then global deps for relative commands.  In my case, I think removing the explicit path prefix helped with running it on FreeBSD without docker.  I don't remember the exact reason right this moment, this was a while back, but I think I had to custom compile one of the deps because it was broken on FreeBSD and I installed the result globally (I think that was the case, but don't count me on it).
